src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageTabs.tsx

```diff
@@ -83,6 +84,16 @@ const DataBlockPageTabs = ({
       releaseVersionId,
     );
 
+    if (dataBlock.charts[0]?.type === 'map') {
```

Comment:
Nice one!

I was trying to find a way to see if it could be done without having to reassign the value of `tableData.subjectMeta.locations` but rather supply it upfront, as per examples like:

```
return mapFullTable({
      ...tableData,
      subjectMeta: {
        ...tableData.subjectMeta,
        locations:
          selectedBoundaryLevel && geoJson
            ? geoJson
            : tableData.subjectMeta.locations,
      },
    });
```

but I couldn't, and so gave up!

Reply:
Like so? :D